qa/sys/centos/bootstrap.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+yum update
+yum install -y java-1.8.0-openjdk-devel.x86_64
+
+##
+# Install logstash manually from a URL
+##
+BRANCH=${LOGSTASH_BRANCH:-'master'}
+BUILD_URL='https://s3-eu-west-1.amazonaws.com/build-eu.elasticsearch.org/logstash'
+URL="$BUILD_URL/$BRANCH/nightly/JDK8/logstash-latest-SNAPSHOT.rpm"
+wget --no-verbose $URL
```

Comment:
I wonder why this have to happen in the privileged provision script (run under root), any idea? I would move them to be in the user_bootstrap.

on the other side, what is the benefit from using the snapshot or simply using the local shared directory? with the latest we're testing with 1 less download. What do you think?